src/Classes/Canvas.ts

```diff
@@ -69,53 +89,36 @@ export class Canvas {
         this.paint();
     }
 
-    /**
-     * Calculates the size of the canvas
-     */
-    public size(): void {
-        const container = this.container;
-        const options = container.options;
-
-        this.element.width = this.dimension.width;
-        this.element.height = this.dimension.height;
-
-        if (options.interactivity.events.resize) {
-            window.addEventListener("resize", () => {
-                this.dimension.width = this.element.offsetWidth;
-                this.dimension.height = this.element.offsetHeight;
-
-                /* resize canvas */
-                if (container.retina.isRetina) {
-                    this.dimension.width *= this.pxRatio;
-                    this.dimension.height *= this.pxRatio;
-                }
-
-                this.element.width = this.dimension.width;
-                this.element.height = this.dimension.height;
+    public changeCanvas(canvas: HTMLCanvasElement) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'changeCanvas' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5179066)